### PR TITLE
REL: v0.1.45

### PR DESCRIPTION
**Pull request by andyfaff**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by andyfaff** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,10 +14,11 @@ source:
 build:
   number: 0
 
-  skip: true  # [py==27, py<39]
+  skip: true  # [py==27, py<310]
```
Comment (jaimergp) [inline recipe/meta.yaml]: This syntax is not correct. I think you mean just:

```suggestion
  skip: true  # [py<310]
```

Comment (andyfaff) [reply recipe/meta.yaml]: Ok. I also need some help with the re-render process. WHen I rerender the recipe the process deletes a whole bunch of necessary files and breaks things. See https://conda.discourse.group/t/conda-forge-re-rendering-of-a-feedstock-removed-necessary-files-in-ci-support/626

**Comment by andyfaff** [comment]
@jaimergp your suggested addition didn't work unfortunately, a whole bunch of CI runs started failing.

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipe/meta.yaml]: It was rerendering to no files because the syntax was wrong. A rerender now is working correctly.

**Comment by andyfaff** [comment]
Thank you for helping @jaimergp

**Pull request merged**